Docs: Move `-params-file` config from CLI to config (#5534)
nextflow-io/nextflow: docs/config.md

```

See {ref}`workflow-handlers` for more information.

(config-params-file)=

## Params file

Parameters specified on the command line can be specified in a params file using the `-params-file` option.

```bash
nextflow run main.nf -params-file pipeline_params.yml
```

The `-params-file` option loads parameters for your Nextflow pipeline from a JSON or YAML file. Parameters defined in the file are equivalent to specifying them directly on the command line. For example:

```bash
nextflow run main.nf --alpha 1 --beta foo
```

> The above parameters can be represented in YAML format:

Reviewer: Feels like this doesn't quite follow on / flow from the text above.
```suggestion
would be equivalent to the following in YAML format:
```

Author: It was feeling uneasy for me too. New flow proposed.
